src/fem/fe_values.hh

```diff
@@ -149,6 +149,13 @@ public:
 //      */
 //     std::vector<std::vector<arma::mat::fixed<spacedim,spacedim> > > shape_grad_vectors;
 
+    /**
+     * @brief Divergence of shape functions (for vectorial finite elements).
+     * 
+     * TODO: compute div from grads; (is not trivial in rt xfem)
+     */
+    std::vector<std::vector<double> > shape_divergence;
```

Comment:
Not clear what is stored in vectors. Neither why this is necessary for XFEM in contradiction to  FEM.